src/assets/models/index.js

```diff
@@ -3,6 +3,7 @@
 const fs = require('fs');
 const path = require('path');
 const Sequelize = require('sequelize');
+const process = require('process');
```

Comment:
Why is this not imported like the other files?

Reply:
Because I thought it's not a good approach when you mix different import styles, but if you want a can change all `require -> import`.

Reply:
Good point, let's leave this for a future refactor (which probably will happen when we work on the v7 version in TypeScript)